andela/coding-geeks-ah-backend
# #168247988 Refactor route definitions according to resources

### What does this PR do?
Groups route definitions according to their respective resources.

### How should this be manually tested?
- `git fetch`
- `git checkout ch-refactor-route-definitions-168247988`

- `npm install`

- `npm run dev`

-  Using postman test using the following endpoints
    ``` 
    - Run any endpoint and observe if the required information will be displayed.
  
### What are the relevant pivotal tracker stories?
[Refactor route definitions](https://www.pivotaltracker.com/story/show/168247988)

### Screenshots
N/A
